config/routes.rb

```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :greetings, only: [:index]
```

Comment:
- [ ] When I try to fetch the greeting. I get this CORS policy error. Kindly fix that. You can visit the [CORS gem](https://github.com/cyu/rack-cors) and configure your backend to allow cross-origin requests.

  Basically, you need to:
  - Uncomment `gem 'rack-cors'` in your Gemfile and run `bundle install`
  - In the `config/initializers/cors.rb` file and uncomment the following code and remember to change "example.com" to "*":
    
    ```
    Rails.application.config.middleware.insert_before 0, Rack::Cors do
      allow do
        origins '*'
        resource '*', headers: :any, methods: [:get, :post, :patch, :put]
      end
    end
    ```
  - Restart your server

  ![image](https://user-images.githubusercontent.com/36542986/203308033-9a028568-6306-481a-8436-a7ff95145ec5.png)

Reply:
I will implement it accordingly.